contracts/Pair.sol

```diff
@@ -148,45 +243,88 @@ contract Pair is IPair {
         }
     }
 
+    // dunks update0
+
     // Accrue fees on token0
-    function _update0(uint amount) internal {
-        _safeTransfer(token0, fees, amount); // transfer the fees out to PairFees
-        uint256 _ratio = amount * 1e18 / totalSupply; // 1e18 adjustment is removed during claim
-        if (_ratio > 0) {
-            index0 += _ratio;
+    // function _update0(uint256 amount) internal {
+    //     _safeTransfer(token0, fees, amount); // transfer the fees out to PairFees
+    //     uint256 _ratio = (amount * 1e18) / totalSupply; // 1e18 adjustment is removed during claim
+    //     if (_ratio > 0) {
+    //         index0 += _ratio;
+    //     }
+    //     emit Fees(msg.sender, amount, 0);
+    // }
+
+    // // Accrue fees on token1
+    // function _update1(uint256 amount) internal {
+    //     _safeTransfer(token1, fees, amount);
+    //     uint256 _ratio = (amount * 1e18) / totalSupply;
+    //     if (_ratio > 0) {
+    //         index1 += _ratio;
+    //     }
+    //     emit Fees(msg.sender, 0, amount);
+    // }
+
+    // Accrue fees on token0
+    function _update0(uint256 amount) internal {
+        if (hasGauge == false) {
```

Comment:
@duncan4123 you can just say `if (!hasGauge)` for false and `if (hasGauge)` for true

Reply:
or `if (hasGauge) { ... } else { ... }`